Gemfile.lock

```diff
@@ -27,47 +27,31 @@ PATH
 GEM
```

Comment:
Should this file be committed to the repo?

I always add *.lock to .gitignore to prevent accidentally committing.